va/va.go

```diff
@@ -575,60 +552,36 @@ func (va *ValidationAuthorityImpl) processRemoteValidationResults(
 		} else {
 			bad++
 		}
-		// Store the first non-nil problem to return later (if `MultiVAFullResults`
-		// is enabled).
+		// Store the first non-nil problem to return later.
 		if firstProb == nil && result.Problem != nil {
 			firstProb = result.Problem
 		}
-		// If MultiVAFullResults isn't enabled then return early whenever the
-		// success or failure threshold is met.
-		if !features.Get().MultiVAFullResults {
-			if good >= required {
-				state = "success"
-				return nil
-			} else if bad > va.maxRemoteFailures {
-				modifiedProblem := *result.Problem
-				modifiedProblem.Detail = "During secondary validation: " + firstProb.Detail
-				return &modifiedProblem
-			}
+		// Return as soon as we have enough successes or failures for a definitive result.
+		if good >= required {
+			state = "success"
+			return nil
+		} else if bad > va.maxRemoteFailures {
+			modifiedProblem := *result.Problem
+			modifiedProblem.Detail = "During secondary validation: " + firstProb.Detail
+			return &modifiedProblem
 		}
 
-		// If we haven't returned early because of MultiVAFullResults being enabled
-		// we need to break the loop once all of the VAs have returned a result.
+		// If we somehow haven't returned early, we need to break the loop once all
+		// of the VAs have returned a result.
```

Comment:
A further available simplification: the `return probs.ServerInternal("Too few remote PerformValidation RPC results")` line below can be inlined here, in place of the `break`.